### ISSUE TITLE
Reply-To Name not correctly interpreted

### ISSUE THREAD

**Issue by Af0x** [osTicket/osTicket#5718]
Hello,

my Reply-To header looks like this:

`Reply-To: My Name - <mail@domain.tld>`

I have the filter for using the Reply-To header active.

The mail-address gets used correctly but the name gets cut off like so:

> My -

This name is used on several places like the frontend ticket view and the backend ticket view. Interestingly when I klick on the name in the backend the name including the minus symbol is shown but when I remove the minus manually the full name is shown in the erroneous fields from before.

Best Regards,

Af0x

**Comment by JediKev** [comment]
@Af0x 

Where is the Server Information, osTicket Version, etc.? Also, hare you fetching or piping the mail?

Cheers.

**Comment by Af0x** [comment]
Sorry, osTicket v1.15, Apache2, PHP 7.3.19, MySQL 5.5.5-10.3.25-MariaDB, Piping Mail.

**Comment by Af0x** [comment]
I think you can reproduce it by just editing a random user name and adding a space and a minus at the end.

**Comment by JediKev** [comment]
@Af0x 

I'm unable to replicate this by simply adding space + `-` to the end of a User's name. I will attempt to pipe an email and see if I can replicate it like that.

Cheers.

**Comment by JediKev** [comment]
@Af0x 

I think I know what's going on now. I bet you have the **Name Format** for Users set to `First Last`, right?

If so, you aren't seeing the Full Name because we break apart the name by spaces. So, if the name is `My Name -` the First Name would be `My` the Middle Name would be `Name` and the Last Name would be `-`. If you want to show the Full User's Name, then you should set **Admin Panel > Settings > Users > Settings > Name Format** to `-- As Entered --`.

Cheers.

**Comment by Af0x** [comment]
You are absolutely right, thank you so much! I guess there is no way to remove this minus with the filter function as I could not find an action with this purpose. Best regards

**Comment by ntozier** [comment]
Out of vague curiosity, isn't the proper format of the Reply To header actually:
"John Doe <john.doe@domain.com>"

I'm not super versed on the syntax of RFC 822 (and other email RFCs) but that was my understanding.
Also I'm pretty sure that osTicket parses anything between `"John Doe <` as the users name. 
So if you are putting in `"John Doe - <` then it literally thinks that `-` is the last name.

**Comment by Af0x** [comment]
I don´t know but I opened an issue for removing it :-)
